notebooks/HASP/WavelengthAdjustment/WavelengthAdjustment.ipynb

```diff
@@ -0,0 +1,2632 @@
+{
```

Comment:
<p>The cos_data is already a path object we can leverage pathlib for that:</p>

<pre class="ql-syntax" spellcheck="false">for file in cos_data.glob("*x1d*"):
&nbsp;&nbsp;shutil.move(str(file), cos_unshifted / file.name)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>